src/analysis/explorerHomeView.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import {CredGraph} from "../core/credrank/credGraph";
+import {type TimestampMs} from "../util/timestamp";
+import {type Interval} from "../core/interval";
+import * as Grain from "../core/ledger/grain";
+
+export type DataOverTime = $ReadOnlyArray<number>;
+
+export type UserRow = {|
+  +name: string,
+  +credEarnedInTimeRange: number,
+  +grainHarvestedInTimeRange: Grain.Grain,
+  +grainBalance: Grain.Grain,
+  +allTimeCred: DataOverTime,
+  +category: UserCategory,
+|};
+
+export type UserCategory = "participant" | "bot" | "project" | "organization";
```

Comment:
Use the `IdentityType` type:

`import type {IdentityType} from "../core/identity"`

Reply:
there we go. accepted.